### ISSUE TITLE
Cannot disable "word highlighting", no setting for it

### ISSUE THREAD

**Issue by Yayuro** [Microsoft/vscode#17341]
- VSCode Version: 1.8.0
- OS Version: macOS 10.12.1

Steps to Reproduce:

1. Open preferences: cmd + ,
2. Change: editor.selectionHighlight to false & save
3. Open any file
4. Click on any repeated function/var

![screen shot 2016-12-15 at 20 01 39](https://cloud.githubusercontent.com/assets/9949969/21240014/5db13258-c301-11e6-9d25-72f138eeefba.png)

It was like two that versions back as well.

Cheers,
Greg

**Comment by rebornix** [comment]
I can reproduce this issue and find a good case and a bad one 

```
const fetchData = () => {

};

fetchData();

var a = 0;
var b = 0;
```

Selecting `var` wont' trigger matching words highlighting while selecting `fetchData` will do.

**Comment by rebornix** [comment]
The issue that users get confused with word highlight and selection matching highlight is already tracked in https://github.com/Microsoft/vscode/issues/2381 . cc @bgashler1

**Comment by Yayuro** [comment]
Thx for reply.
I didn't mean var as 'var' word but name of defined var - same as defined function (fetchData in this case).
So color is one thing, but is there any way right now to turn it off completely?

Thx,
Greg

**Comment by stormherz** [comment]
Was wondering the same thing - is it possible to disable this behaviour?

**Comment by alexandrudima** [comment]
Fixed in #5351 - `editor.occurrencesHighlight`
